Detectors/TPC/workflow/include/TPCWorkflow/RecoWorkflow.h

```diff
@@ -78,7 +78,8 @@ framework::WorkflowSpec getWorkflow(CompletionPolicyData* policyData,
                                     int zsOnTheFly = 0,
                                     int zs10bit = 0,
                                     float zsThreshold = 2.0f,
-                                    bool askDISTSTF = true);
+                                    bool askDISTSTF = true,
+                                    std::string const& nativeClusterFilename = "tpc-native-clusters.root");
```

Comment:
Currently the function to produce the workflow spec takes a long list of arguments. I think it would make sense
to encapsulate everything into a TPCRecoConfig struct instead. This can be coupled to the workflow program options and would make future modifications easier.